$MirOS: ports/www/firesomething/patches/patch-nsprpub_pr_include_md__openbsd_h,v 1.3 2008/05/21 06:31:04 bsiegert Exp $
$OpenBSD: patch-nsprpub_pr_include_md__openbsd_h,v 1.7 2005/04/09 15:43:47 wilfried Exp $
--- nsprpub/pr/include/md/_openbsd.h.orig	2007-05-26 02:13:08.000000000 +0200
+++ nsprpub/pr/include/md/_openbsd.h	2008-08-24 16:10:05.000000000 +0200
@@ -58,7 +58,7 @@
 #define _PR_SI_ARCHITECTURE "arm"
 #endif
 
-#define PR_DLL_SUFFIX		".so"
+#define PR_DLL_SUFFIX		".so._SO_VERSION_"
 
 #define _PR_VMBASE              0x30000000
 #define _PR_STACK_VMBASE	0x50000000
